assets/js/wpuf-ajax-login.js

```diff
@@ -0,0 +1,81 @@
+jQuery(function($){
+
+    $('#wpuf-ajax-reset-password').hide();
+    // Switch forms login/register
+    $('a[href="#wpuf-ajax-login-url"]').click(function(e){
```

Comment:
Instead of adding individual events to links, you should use the most top div class or id. e.g. `$('.wpuf-ajax-login-form')`

```js
var form = $('.wpuf-ajax-login-form');

form.on('click', '$('a[href="#wpuf-ajax-login-url"]')', function(e) {
    e.preventDefault();
    
    // code here    
});

form.on('submit', 'form#wpuf_ajax_login_form', function(e) {
    e.preventDefault();

    // code here
});

```